client/jetpack-connect/auth-logged-in-form.jsx

```diff
@@ -118,7 +116,7 @@ class LoggedInForm extends Component {
 
 		// isSSO is a separate case from the rest since we have already validated
 		// it in authorize-form.jsx. Therefore, if it's set, just authorize and redirect.
-		if ( this.props.isSSO || doAutoAuthorize ) {
+		if ( this.isSso() || doAutoAuthorize ) {
```

Comment:
We're not passing props to `isSso` here. I wonder if that's the issue? I'm not super familiar with the operator we're using in the `isSso` method below.

Reply:
Maybe being too fancy there 😓 

https://github.com/Automattic/wp-calypso/pull/19956/files#diff-e8644e2359a41377a490f6cbe8003db3R186

```
isSso( { from, queryDataSiteId } = this.props )
```

* `{ from, queryDataSiteId }` -> destructuring assignment
* `... = this.props` -> default argument

We should be assigning `from` and `queryDataSiteId` from the properties on the first argument supplied, and if that argument is not supplied (`undefinted`), then we should default to using `this.props` as the argument.

In theory this should work, but I'll need to verify.